README.md

```diff
@@ -23,6 +23,10 @@ nmsg has the following external dependencies:
 
 * [zlib](http://www.zlib.net/)
 
+* prometheus integration (optional)
+  * [libprometheus](https://gitlab.vm.domaintools.net/debian/prometheus-client-c)
```

Comment:
I asked in team-rt-eng is our fork is different than the official upstream.  If it's the same, then maybe point to the upstream rather than our internal fork